control-plane-operator/controllers/hostedcontrolplane/etcd/reconcile.go

```diff
@@ -166,14 +166,12 @@ func ReconcileStatefulSet(ss *appsv1.StatefulSet, p *EtcdParams) error {
 
 func buildEtcdInitContainer(p *EtcdParams) func(c *corev1.Container) {
 	return func(c *corev1.Container) {
-		c.Env = []corev1.EnvVar{}
-		for i := 0; i < p.DeploymentConfig.Replicas; i++ {
-			c.Env = append(c.Env, corev1.EnvVar{
-				Name:  fmt.Sprintf("RESTORE_URL_ETCD_%d", i),
-				Value: p.StorageSpec.RestoreSnapshotURL[i],
-			})
+		c.Env = []corev1.EnvVar{
+			{
+				Name:  "RESTORE_URL_ETCD",
+				Value: p.StorageSpec.RestoreSnapshotURL[0], // RestoreSnapshotURL can only hve 1 entry
```

Comment:
- What happens if we have multiples ETCD pods (HA)?
- The backup will take only one and upload that one or what will be the behaviour?

Reply:
there is no behavior change either in HA or normal mode. We just need a single snapshot for all etcd members, and this PR enforces that through the API.